src/concern/mod.rs

```diff
@@ -152,8 +175,9 @@ impl Acknowledgment {
 }
 
 impl WriteConcern {
+    #[allow(dead_code)]
```

Comment:
I don't think this is dead code; it gets called in `Operation::is_acknowledged`, which isn't marked as dead code

Reply:
Yeah this one is a little weird. The compiled doesn't complain about `Operation::is_acknowledged` being unused (maybe because it's part of a trait definition?), but it does complain about this method.

Reply:
Hmm. Is there any reason to bother keeping them both around then, if neither is used?

Reply:
Oh, wait, is it needed for sessions?

Reply:
Yep, they're just dead code right now because this is a separate PR. When I rebase the other one I can remove the ignore.

Reply:
Awesome